server/config/baseUrlServer.xml

```diff
@@ -44,7 +44,7 @@ limitations under the License.
          same MongoDB driver classes.
       -->
     <library id="mongo-lib" apiTypeVisibility="spec,ibm-api,api,third-party">
-        <file name="${shared.resource.dir}/libs/mongo-java-driver-3.11.0.jar"/>
+        <fileset dir="${shared.resource.dir}/libs" includes="*.jar" />
```

Comment:
I wonder if this should be includes mongo-java-driver*.jar so that we don't accidentally pick up anything else that may have been placed in the shared resources dir?